### ISSUE TITLE
Can't run test suite

### ISSUE THREAD

**Issue by buxx** [buxx/muzich#2]
When run test suite with command (follow readme to be able to run tests):

    ./phpunit -c app src

All tests are in error. This must be fixed.

**Comment by buxx** [comment]
There is lot of error like:

```
SQLSTATE[42S02]: Base table or view not found: 1146 Table 'muzich_test.m_user' doesn't exist

/muzich/vendor/doctrine/dbal/lib/Doctrine/DBAL/DBALException.php:47
/muzich/vendor/doctrine/dbal/lib/Doctrine/DBAL/Connection.php:649
/muzich/vendor/doctrine/orm/lib/Doctrine/ORM/Persisters/BasicEntityPersister.php:670
/muzich/vendor/doctrine/orm/lib/Doctrine/ORM/EntityRepository.php:171
/muzich/vendor/doctrine/orm/lib/Doctrine/ORM/EntityRepository.php:211
/muzich/src/Muzich/CoreBundle/Tests/ElementFactory/ElementFactoryTest.php:23
/muzich/src/Muzich/CoreBundle/Tests/ElementFactory/ElementFactoryTest.php:23
```

It look like fixtures are missing ...

**Comment by buxx** [comment]
Correct way to prepare test database:

* ```php app/console doctrine:database:create --env=test -n```
* ```php app/console doctrine:schema:create --env=test -n```

But when execute ```php app/console doctrine:fixtures:load --env=test -n```:

```
  [Gedmo\Exception\UnsupportedObjectManagerException]  
  Given object manager is not managed by wrapper
```

**Comment by buxx** [comment]
Exception solved with 388d065. Have now to fix tests errors.
